usecases/schema/class.go

```diff
@@ -151,61 +150,33 @@ func (h *Handler) UpdateClass(ctx context.Context, principal *models.Principal,
 	className string, updated *models.Class,
 ) error {
 	err := h.Authorizer.Authorize(principal, "update", "schema/objects")
-	if err != nil {
-		return err
-	}
-
-	initial := h.metaReader.ReadOnlyClass(className)
-	if initial == nil {
-		return ErrNotFound
-	}
-	mtEnabled, err := validateUpdatingMT(initial, updated)
-	if err != nil {
+	if err != nil || updated == nil {
```

Comment:
**thought (non-blocking):**  for `updated == nil`, i think err would be nil, do we want to return specific error if updated is nil ?

Reply:
I believe `return nil if updated == nil`, which means no updates, is acceptable. 
In practice` updated==nil` never happens